rpgsaga/saga/src/Person.ts

```diff
@@ -0,0 +1,37 @@
+export abstract class Person {
+  private _name: string;
+  private _age: number;
+  private _sick: boolean;
+
+  constructor(name: string, age: number, isSick: boolean = false) {
+    this._name = name;
+    this.age = age;
+    this._sick = isSick;
```

Comment:
```suggestion
    this.name = name;
    this.age = age;
    this.sick = isSick;
```

таким образом мы переиспользуем сеттеры - и не будет возможно установить параметры в обход сеттеров и их проверок